src/components/dashboard/CustomPieChart.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from "react";
+import { Doughnut } from "react-chartjs-2";
+import {} from "chart.js";
+import "../style/styling.css";
+import "chart.js/auto";
+
+
+const data = {
+    labels: [
+        "Picked Up",
+        "Ready for Pick Up",
+    ],
+    datasets: [{
+        label: "Status",
+        data: [90, 69],
+        backgroundColor: [
```

Comment:
These should be added to the site theme and the accessed using the `useTheme` hook. https://mui.com/material-ui/customization/theming/#accessing-the-theme-in-a-component

Reply:
Implemented the change using a function. However, theme. palette.secondary.main appears as purple